packages/bpk-docs/src/helpers/tokens-helper.js

```diff
@@ -19,32 +19,56 @@
 import union from 'lodash/union';
 import kebabCase from 'lodash/kebabCase';
 
-export const toPx = (value) => {
-  let parsed = null;
-
-  if (/rem$/.test(value)) {
-    parsed = parseFloat(value.replace(/rem/, '')) * 16;
-  }
-
-  if (/%$/.test(value)) {
-    parsed = parseFloat((value.replace(/%/, '')) / 100) * 16;
-  }
+export const formatTokenName = name => kebabCase(name);
 
+export const toPx = (value) => {
+  const parsed = parseFloat(value) * 16;
```

Comment:
`16` is here because that's 1rem right? I'm wondering if that merits a const/comment just to make it totally clear.

Reply:
The best solution would be to have a token for this right? `BASE_FONT_SIZE` or so?

Reply:
Not really - that would mean we introduce a token specifically for converting rem's to pixels in our docs site. It'll be used nowehre else. Plus we already have `FONT_SIZE_ROOT` with is set to `100%` (see https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/src/web/base/typography.json#L41)

Reply:
But this has a relation to all our rem values. If we change the base font size to say 18px all our rem values have to be recalculated

Reply:
If we do do that (which I doubt we will) then we'll notice that all the values are incorrect in the docs and we'll update this constant

Reply:
Fair enough then :)